recipes/model_catalogs/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "model_catalogs" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/model_catalogs-{{ version }}.tar.gz
+  sha256: f32632e605bc0803265483cc7c68ccedaeba6195165b0e6d10b0f424c7de8b8b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - pandas
+    - netcdf4
+    - intake-xarray
+    - siphon
+    - dask-core
+    - intake
+    - appdirs
+    - cf_xarray
+    - xarray
+    - pip
+    - datetimerange
+
+test:
+  imports:
+    - model_catalogs
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/NOAA-ORR-ERD/model_catalogs
+  summary: Catalogs for known models
+  license: MIT
+  license_file:
+    - LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - kthyng
+    - ChrisBarker-NOAA
```

Comment:
All maintainers need to comment saying they are ok being on the recipe. @ChrisBarker-NOAA?

Reply:
Oops, sorry I hadn't had that happen in the meantime!

Reply:
@beckermr Please see note from @ChrisBarker-NOAA below.